packages/editor/src/components/post-preview-button/index.js

```diff
@@ -98,10 +98,6 @@ export class PostPreviewButton extends Component {
 		// unintentional forceful redirects.
 		if ( previewLink && ! prevProps.previewLink ) {
 			this.setPreviewWindowLink( previewLink );
-
-			// Once popup redirect is evaluated, even if already closed, delete
-			// reference to avoid later assignment of location in post update.
-			delete this.previewWindow;
```

Comment:
@aduth I removed this to be able to refresh the window without an autosave (once the metaboxes finish saving). I'm not aware of any downsides but let me know if I missed something.